docs/source/terms_and_model.rst

```diff
@@ -379,9 +375,49 @@ A contiguous region specified by named features.
    {
      'end': 'q22.3',
      'start': 'q22.2',
-     'type': 'NamedInterval'
+     'type': 'CytobandInterval'
```

Comment:
In other examples, the key and values are enclosed in double quotes e.g "end" : "q22.3"
The ChromosomalLocation example also has single quotes.